vividus-bdd-engine/src/main/java/org/vividus/bdd/expression/FunctionalExpressionProcessor.java

```diff
@@ -17,25 +17,25 @@
 package org.vividus.bdd.expression;
 
 import java.util.Optional;
-import java.util.function.UnaryOperator;
+import java.util.function.Function;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class UnaryExpressionProcessor implements IExpressionProcessor
+public class FunctionalExpressionProcessor implements IExpressionProcessor
```

Comment:
I would kept `UnaryExpressionProcessor` as a child of `FunctionalExpressionProcessor `